package.json

```diff
@@ -1,35 +1,44 @@
 {
   "name": "agent-api-simulator",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "Agent Api Simulator",
   "author": "Genesys",
   "license": "MIT",
   "repository": {
     "type": "git",
     "url": "https://github.com/GenesysPureEngage/agent-api-simulator"
   },
+  "workspace-development-kit": {
+    "scapi-samples": {
+      "dir": "https://genesyspureengage.github.io/workspace-development-kit/wwe-service-client-api/api-overview/",
+      "files": ["sample.html", "wwe-service-client-api.js"]
+    }
+  },
```

Comment:
Workspace Development Kit samples, are defined directly in package.json.
Then, this package.json is read from src/scripts/downloadUiAssets.js (_was done before_) script in order to retrieve the assets.